sklearnex/basic_statistics/tests/test_basic_statistics.py

```diff
@@ -178,6 +181,55 @@ def test_multiple_options_on_random_data(
     assert_allclose(gtr_sum, res_sum, atol=tol)
 
 
+@pytest.mark.parametrize("queue", get_queues())
+@pytest.mark.parametrize("row_count", [100, 1000])
+@pytest.mark.parametrize("column_count", [10, 100])
+@pytest.mark.parametrize("weighted", [True, False])
+@pytest.mark.parametrize("dtype", [np.float32, np.float64])
+def test_multiple_options_on_random_sparse_data(
+    queue, row_count, column_count, weighted, dtype
+):
+    seed = 77
+    random_state = 42
+    gen = np.random.default_rng(seed)
+    X, _ = make_blobs(
+        n_samples=row_count, n_features=column_count, random_state=random_state
+    )
+    density = 0.05
+    X_sparse = csr_matrix(X * (np.random.rand(*X.shape) < density))
+    X_dense = X_sparse.toarray()
+
+    if weighted:
+        weights = gen.uniform(low=-0.5, high=1.0, size=row_count)
+        weights = weights.astype(dtype=dtype)
+    basicstat = BasicStatistics(result_options=["mean", "max", "sum"])
```

Comment:
According to onedal tests, need to exclude "max" at it contains bugs:
https://github.com/intel/scikit-learn-intelex/blob/main/onedal/basic_statistics/tests/test_basic_statistics.py#L273

Reply:
> According to onedal tests, need to exclude "max" at it contains bugs: https://github.com/intel/scikit-learn-intelex/blob/main/onedal/basic_statistics/tests/test_basic_statistics.py#L273

@olegkkruglov please message out a link to the ticket associated with this error (just to make sure it wasn't lost)

Reply:
Sorry, my mistake, I didn't dig deep enough in the git blame.  Turns out it was introduced here https://github.com/intel/scikit-learn-intelex/pull/1846 by @Vika-F .  Do you know if there was any follow-up work after #1846 on the max issues/ any memory on what was going on?

Reply:
I observed that issue as well, temporarily removed the "max" in tests.